reviews/2021-openservicemesh-annual.md

```diff
@@ -0,0 +1,82 @@
+# Open Service Mesh (OSM) Annual Review 2021
+
+## Project Overview
+
+**[Open Service Mesh](https://openservicemesh.io)** is a simple, lightweight, extensible cloud native service mesh that allows users to uniformly manage, secure, and get out-of-the-box observability features for highly dynamic microservice environments.
+
+## DevStats
+
+> Include a link to your project’s devstats page. We will be looking for signs of consistent or increasing contribution activity. Please feel free to add commentary to add colour to the numbers and graphs we will see on devstats.
+
+Open Service Mesh DevStats can be found [here](https://openservicemesh.devstats.cncf.io/d/8/dashboards?orgId=1&from=now-365d&to=now-1h).
+
+## Maintainers
+
+> How many maintainers do you have, and which organisations are they from? (Feel free to link to an existing MAINTAINERS file if appropriate.)
+
+OSM currently has 15 [maintainers](https://github.com/openservicemesh/osm/blob/main/OWNERS).
+
+- Delyan Raychev, Microsoft
+- Eduardo Serra Miralles, VMware
+- Kalya Subramanian, Microsoft
+- Michelle Noorali, Independent
+- Jon Huhn, Microsoft
+- Shashank Ram, Microsoft
+- Sanya Kochhar, Microsoft
+- Shalier Xia, Microsoft
+- Niranjan Shankar, Microsoft
+- Sneha Chhabria, Microsoft
+- Phillip Gibson, Microsoft
+- Jackie Elliot, Microsoft
+- Thomas Stringer, Microsoft
+- Allen Leigh, Microsoft
+- Rita Zhang, Microsoft
+
+Beyond the maintainers list, OSM has had 64 unique [contributors](https://github.com/openservicemesh/osm/graphs/contributors) to the project. Over the past year there have been [10 contributing companies and up to 48 contributing developers](https://openservicemesh.devstats.cncf.io/d/7/companies-contributing-in-repository-groups?orgId=1&from=now-365d&to=now-1h&var-period=d28&var-repogroup_name=All).
+
+## Adoption
+
+> What do you know about adoption, and how has this changed since your last review / since you joined Sandbox? If you can list companies that are end users of your project, please do so. (Feel free to link to an existing ADOPTERS file if appropriate.)
+
+Strong end-user adoption has grown since the OSM project entered into the sandbox stage. This adoption has made the OSM project pivot to supporting additional community projects collaborations/integrations such as [Flagger](https://github.com/fluxcd/flagger/blob/main/docs/gitbook/tutorials/osm-progressive-delivery.md), [Contour Ingress](https://release-v0-11.docs.openservicemesh.io/docs/demos/ingress_contour/), and [Open Policy Agent](https://release-v0-11.docs.openservicemesh.io/docs/guides/integrations/external_auth_opa/), with more in the works.
```

Comment:
```suggestion
Strong end-user adoption has grown since the OSM project entered into the sandbox stage. This adoption has made the OSM project pivot to supporting additional community projects collaborations/integrations such as [Flagger](https://github.com/fluxcd/flagger/blob/main/docs/gitbook/tutorials/osm-progressive-delivery.md), [Contour Ingress](https://release-v0-11.docs.openservicemesh.io/docs/demos/ingress_contour/), [Meshery](https://docs.meshery.io/service-meshes/adapters/osm) and [Open Policy Agent](https://release-v0-11.docs.openservicemesh.io/docs/guides/integrations/external_auth_opa/), with more in the works.
```

Reply:
@phillipgibson, you might add OSM's integration with Meshery to the list.